HIP/hip-583.md

```diff
@@ -2,221 +2,268 @@
+  transaction being submitted.
+- `Hollow account` - a partially created account which lacks a public key but does have a valid `shard.realm.num`
+  accountId _and_ is auto-created with an `evm address alias`. Such an account may receive assets but may not take
+  part in transactions that require its signature, until it is promoted to being a "full" account.
+- `lazy-create` - a unique account creation flow in which the public address (extracted from a provided ECDSA public
+  key) is used to automatically create a hollow account without an explict `CryptoCreate` transaction being submitted.
+
+With an Ethereum-native wallet, you can transfer the native currency (in our case, HBAR), or tokens, to an `evm address`
+that does not yet exist. If nobody in the universe has the private key for that account, the value held by that account
+is effectively locked forever. If somebody does have the keys to that account, they can access the account and control
+it. Since Ethereum-native wallets do not understand the `0.0.10` style format of a Hedera account ID, it is necessary
+to support the creation of accounts with an `evm address alias` to enable compatibility between Ethereum-native
+wallets and Hedera-native wallets and all accounts.
+
+Note that this HIP does not add support for adding an `alias` to an existing account. See
+[HIP 631](https://hips.hedera.com/hip/hip-631) for a proposal for adding lifecycle for the account `alias`.
 
 ## User stories
 
-1. As a user with an `ECDSA` based account from another chain I would like to create my Hedera account using my `public-address`.
-
-2. As a developer, I would like to `auto-create` a new account using a public key via the `CryptoCreate` transaction.
-
-3. As a developer, I would like to `lazy-create` a new account using a `public-address` via the `CryptoCreate` transaction.
-
-4. As a developer, I would like to `lazy-create` a new account using a `public-address` via the  `CryptoTransfer` transaction.
+1. As a user with an `ECDSA` derived account from another chain, I would like to create my Hedera account using that
+   same ECDSA public key and associated Ethereum compatible `evm address alias`.
+2. As a developer, I would like to create a new account with a `key alias` via the `CryptoCreate` transaction.
+3. As a developer, I would like to create a new account with an `evm address alias` via the `CryptoCreate` transaction.
+4. As a developer, I would like to transfer HBAR or tokens to a Hedera account using their `evm address alias`.
+5. As a Hedera user with an `Ethereum-native wallet`, I would like to receive HBAR or tokens in my account by sharing
+   only my `evm address alias`.
+6. As a Hedera user with a `Hedera-native wallet`,  I would like to transfer HBAR or tokens to another account using
+   only the recipient's `evm address alias`.
+7. As a Hedera user with an `ED25519` key on my account, I want to add an `evm address alias` to my account.
```

Comment:
Q: To clarify this is a new feature for HIP 583.
Essentially CryptoUpdate and pass in an evm address alias to unblock accounts with no evm address alias in state or memory?

Reply:
Yes. I wasn't going to put this into the HIP, until I saw that HIP-32 already has language for supporting alias on account update! So we *have* to have this language in this HIP to clarify what happens on account update, since HIP-32 already mandates it (even if we haven't implemented it yet).

Reply:
OK, I switched direction on this. I decided to **supersede** this language from HIP-32 to indicate that we do NOT support adding an alias to an existing account, with a section at the bottom explaining why.